src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -193,12 +105,6 @@ public static function debug_data() {
 			'fields' => array(),
 		);
 
-		$info['wp-themes-inactive'] = array(
-			'label'      => __( 'Inactive Themes' ),
-			'show_count' => true,
-			'fields'     => array(),
-		);
```

Comment:
looks like this other change is still in here

Reply:
solved.